packages/vm/tests/accountexists.spec.ts

```diff
@@ -0,0 +1,45 @@
+import Common, { Chain, Hardfork } from '@ethereumjs/common'
+import { Address, BN } from 'ethereumjs-util'
+import tape from 'tape'
+import VM from '../src'
+
+tape('correctly apply new account gas fee on pre-Spurious Dragon hardforks', async (t) => {
+  // setup the accounts for this test
```

Comment:
For completeness of this test and some background info I think it would be nice if we can add the txhash of this specific test.

Reply:
Review the comments I added and let me know if this is clear enough.

Reply:
Done!